src/modules/dashboard/components/ColonyHome/ColonySubscription/ColonySubscription.css

```diff
@@ -51,13 +68,24 @@
   left: 0;
 }
 
-.createUserRedirect {
-  font-size: var(--size-tiny);
+.joinColonyBtn {
```

Comment:
nitpicky... but.... for the naming convention, until now the names started with 'colony' (i.e. colonySubscribed, colonyJoin) but here you switch to joinColony(Btn).

Same in the .tsx file.